rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,6 +16,8 @@ toc::[]
 [id="rosa-q4-2024_{context}"]
 === Q4 2024
 
+* *Learning tutorials for ROSA cluster and application deployment.* You can now use the xref:../cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-choose-deployment-method.adoc#cloud-experts-getting-started-choose-deployment-method[Getting started with ROSA] tutorials to quickly deploy a ROSA cluster for demo or learning purposes. You can also use the xref:../cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc#cloud-experts-deploying-application-intro[Deploying an application] tutorials to deploy an application on your demo cluster.
```

Comment:
```suggestion
* *Learning tutorials for ROSA cluster and application deployment.* You can now use the xref:../cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-choose-deployment-method.adoc#cloud-experts-getting-started-choose-deployment-method[Getting started with ROSA] tutorials to quickly deploy a ROSA cluster for demo or learning purposes. You can also use the xref:../cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc#cloud-experts-deploying-application-intro[Deploying an application] tutorial to deploy an application on your demo cluster.
```

Reply:
Not sure if this should be singular or not...there are multiple titles(sections) within the "Deploying an application" tutorial but only one "Deploying an application" tutorial...so maybe should be singular? 
This could also be me overthinking it... :(

Reply:
Going to leave it as plural because it is a drop down menu with lots of tutorials.